app/api/resources/mentorship_relation.py

```diff
@@ -658,7 +658,9 @@ class UpdateTask(Resource):
     @mentorship_relation_ns.response(
         HTTPStatus.OK, "%s" % messages.TASK_WAS_ACHIEVED_SUCCESSFULLY
     )
-    @mentorship_relation_ns.response(HTTPStatus.BAD_REQUEST, "%s" % messages.TASK_WAS_ALREADY_ACHIEVED)
+    @mentorship_relation_ns.response(
+        HTTPStatus.CONFLICT, "%s" % messages.TASK_WAS_ALREADY_ACHIEVED
```

Comment:
@divyanshugit to keep the code consistent , can you please remove the added lines and make it into a single one as earlier while the HTTPstatus is updated to CONFLICT

Reply:
Done, as you said. I thought that, If I put the code in the same manner as earlier then it will be easier to read.